exercises/concept/treasure-chest/.meta/design.md

```diff
@@ -0,0 +1,26 @@
+# Design
```

Comment:
Currently there is no need for an analyzer rule that I can think of, but let's add the category to make it explicit.